### DISCUSSION TITLE
Expand docs with Alembic data migration example

### DISCUSSION THREAD

**Discussion by WilliamDEdwards** [fastapi/fastapi#8664] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
-
```

### Description

Alembic can be used for schema migrations. This is documented pretty well. Some migrations involve migrating data (e.g. when adding a column and filling it with data, maybe even from another column).

Although Alembic is not a data migration tool, it can be used as such in certain circumstances. I think it would be useful to add an example of how to do a data migration using Alembic, as this is a relatively common use case.

### Wanted Solution

Expand docs with data migration example.

I'm willing to write this example.

### Wanted Code

```python
-
```

### Operating System

Linux

### FastAPI Version

0.62.0

### Python Version

3.9.5

**Comment by Bastien-BO** [comment, marked as answer]
hello

Since fastapi doesn't come with any ORM (such as sqlalchemy), make it work with alembic or another migration tool as all to do with your ORM/sql toolkit and nothing with fastapi. If you want to add migrations to your fastapi project you can simply folow your ORM and migration tool doc.


An example i use often: [Kludex/fastapi-microservices](https://github.com/Kludex/fastapi-microservices/tree/main/users)

**Comment by WilliamDEdwards** [comment]
> hello
> 
> Since fastapi doesn't come with any ORM (such as sqlalchemy), make it work with alembic or another migration tool as all to do with your ORM/sql toolkit and nothing with fastapi. If you want to add migrations to your fastapi project you can simply folow your ORM and migration tool doc.
> 
> An example i use often: [Kludex/fastapi-microservices](https://github.com/Kludex/fastapi-microservices/tree/main/users)

I'm not sure if I agree. Of course, FastAPI is ORM-independent - but there's plenty of documentation (e.g. at https://fastapi.tiangolo.com/advanced/async-sql-databases/?h=sqlalchemy#import-and-set-up-sqlalchemy) to get users started with general purpose tools such as SQLAlchemy. But maybe this issue is better suited for @tiangolo's boilerplate project.

(Also, I'm not seeing any data migrations in the repo you linked to - but that's not why I created this issue anyway.)
